blocks/editable/index.js

```diff
@@ -36,41 +35,28 @@ import { EVENTS } from './constants';
 
 const { BACKSPACE, DELETE, ENTER } = keycodes;
 
-function createTinyMCEElement( type, props, ...children ) {
-	if ( props[ 'data-mce-bogus' ] === 'all' ) {
-		return null;
-	}
-
-	if ( props.hasOwnProperty( 'data-mce-bogus' ) ) {
-		return children;
-	}
-
-	return createElement(
-		type,
-		omitBy( props, ( value, key ) => key.indexOf( 'data-mce-' ) === 0 ),
-		...children
-	);
-}
-
 function isLinkBoundary( fragment ) {
 	return fragment.childNodes && fragment.childNodes.length === 1 &&
 		fragment.childNodes[ 0 ].nodeName === 'A' && fragment.childNodes[ 0 ].text.length === 1 &&
 		fragment.childNodes[ 0 ].text[ 0 ] === '\uFEFF';
 }
 
+function nodeListToString( nodeList ) {
+	const container = document.createElement( 'div' );
+	nodeList.forEach( ( node ) => container.appendChild( node ) );
+	return container.innerHTML;
+}
+
 export default class Editable extends Component {
 	constructor( props ) {
 		super( ...arguments );
 
 		const { value } = props;
 		if ( 'production' !== process.env.NODE_ENV && undefined !== value &&
-					! Array.isArray( value ) ) {
```

Comment:
With these changes, I think we might be able to just remove this warning altogether.